conda/meta.yaml

```diff
@@ -1,5 +1,5 @@
 {% set name = "e3sm_diags" %}
-{% set version = "2.4.0" %}
+{% set version = "2.5.0rc1" %}
 
```

Comment:
I noticed the `source` section is using source from tarball and not source from git.

This means that when we release on GitHub, that release will contain an outdated SHA reference. 
I'm currently testing this out on `zstash` and `zppy`, but using [source from Git](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#source-from-git) should fix this issue.

Reply:
Also the docs show an example using source from Git.
https://e3sm-project.github.io/e3sm_diags/_build/html/master/dev_guide/releasing-e3sm-diags.html

Reply:
I think at some point, @xylar suggested to use tarball as source. SHA will be then updated after git release:https://e3sm-project.github.io/zstash/_build/html/master/dev_guide/release.html

Reply:
Hmmm, yeah I'm not sure what the benefit is of one over the other.

Using tarbell requires a secondary commit after a release to update the outdated SHA in `meta.yaml`. It also doesn't play nicely with CI/CD, but this won't affect `e3sm_diags` because we won't automate Anaconda publishing to the `e3sm` channel. 

I think we'll be okay with tarbell here.

Reply:
Looks good with the change! One less step every release